src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -573,19 +579,24 @@ void AlgorithmPaintToolBox::activateMagicWand()
 void AlgorithmPaintToolBox::behaveWithBodyVisibility()
 {
     if(!((QWidget*)sender())->isHidden())
-        return;
-    if ( this->m_strokeButton->isChecked() ) {
-        this->m_viewFilter->removeFromAllViews();
-        m_paintState = (PaintState::None);
-        updateButtons();
+    {
         return;
     }
-    if ( this->m_magicWandButton->isChecked() ) {
-        this->m_viewFilter->removeFromAllViews();
-        m_paintState = (PaintState::None);
-        newSeed(); // accept the current growth
-        updateButtons();
-        return;
+    else
+    {
+        if ( this->m_strokeButton->isChecked() )
+        {
+            this->m_viewFilter->removeFromAllViews();
+            m_paintState = (PaintState::None);
+            updateButtons();
+        }
+        else if ( this->m_magicWandButton->isChecked() )
+        {
+            this->m_viewFilter->removeFromAllViews();
+            m_paintState = (PaintState::None);
+            newSeed(); // accept the current growth
+            updateButtons();
+        }
```

Comment:
There's no change here.
My first tries were in this method, so I changed the typo.